tests/components/speedtestdotnet/test_init.py

```diff
@@ -35,13 +38,19 @@ async def test_successful_config_entry(hass):
         await hass.config_entries.async_setup(entry.entry_id)
 
     assert entry.state == config_entries.ENTRY_STATE_LOADED
-    assert forward_entry_setup.mock_calls[0][1] == (entry, "sensor",)
+    assert forward_entry_setup.mock_calls[0][1] == (
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6072450)